code/game/objects/structures/girders.dm

```diff
@@ -171,6 +191,23 @@
 
 		if(!S.has_unique_girder)
 			var/M = S.sheettype
+			if(state == GIRDER_TRAM)
+				if(!S.tram_wall_type)
+					to_chat(user, span_warning("This tram girder only works with Titanium, Plastitanium, or Iron!"))
+					return
+				if(S.get_amount() < 2)
+					to_chat(user, span_warning("You need at least two sheets to add plating!"))
+					return
+				to_chat(user, span_notice("You start adding plating..."))
+				if (do_after(user, 40, target = src))
```

Comment:
use SECONDS defines for all these

Reply:
to be fair, i basically copied false wall code since they act as walls while being structures, only i removed the false wall bits lol.

ill fix up the reviews when i get on my PC later